sample/src/main/java/io/techery/janet/command/sample/CommandSample.java

```diff
@@ -33,6 +36,26 @@ public static void main(String... args) throws Throwable {
 
         actionPipe.send(action);
 
+        janet = new Janet.Builder()
+                .addService(new MockCommandActionService.Builder()
+                        .actionService(new CommandActionService())
+                        .addContract(MockCommandActionService.Contract
```

Comment:
It would be cool if we could set type of `Contract` to ease add function with defined type of argument. 
For example, we can add generic for `Contract` and method e.g. `of(Class<?>)`cast it to the typed one.